src/modules/back-to-top/back-to-top.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../styles/_variables';
+
+.stache-back-to-top {
+  position: fixed;
+  bottom: 10px;
+  right: 10px;
+  color: $stache-color-white;
+  background: $stache-color-blue;
+  border-radius: 5px;
+  height: 30px;
+  width: 30px;
+  text-align: center;
+  font-size: 2em;
+  line-height: 1;
+  opacity: 1;
+  transition: opacity .5s;
+}
+
+.stache-back-to-top:hover,
+.stache-back-to-top:focus {
+  background: $stache-color-link-hover;
+  cursor: pointer;
+}
+
+i.fa {
+  font-size: 16px;
+  display: block;
+  margin-top: 5px;
+}
+
+.hidden {
+  opacity: 0;
```

Comment:
The button is still clickable, even though its opacity is set to zero. Add `display: none`?